.storybook/preview-head.html

```diff
@@ -0,0 +1,9 @@
+<script type="text/javascript">
+  document.addEventListener(
+    'DOMContentLoaded',
+    function () {
```

Comment:
this was specifically done for react comps as consumers of nhsuk-react-components lib - might not be required here as we don't embed this in NHS layout pages